be/src/storage/schema_change.cpp

```diff
@@ -888,10 +1017,83 @@ Status SchemaChangeHandler::_convert_historical_rowsets(SchemaChangeParams& sc_p
                     << ", version=" << sc_params.version.first << "-" << sc_params.version.second;
         }
 
+        std::vector<DeltaColumnGroupList> historical_dcgs;
+        historical_dcgs.resize(sc_params.rowsets_to_change[i]->num_segments());
+        for (uint32_t j = 0; j < sc_params.rowsets_to_change[i]->num_segments(); j++) {
+            int64_t tablet_id = sc_params.rowsets_to_change[i]->rowset_meta()->tablet_id();
+            RowsetId rowsetid = sc_params.rowsets_to_change[i]->rowset_meta()->rowset_id();
+            RETURN_IF_ERROR(TabletMetaManager::get_delta_column_group(new_tablet->data_dir()->get_meta(), tablet_id,
+                                                                      rowsetid, j, INT64_MAX, &historical_dcgs[j]));
+        }
+        if (!sc_params.sc_sorting && !sc_params.sc_directly && chunk_changer->get_mc_exprs()->size() != 0) {
```

Comment:
`gc` not `mc`. `mc` is short for `materialized column`

Reply:
I will fix the naming problem in another pr for both be and fe